src/main/java/de/siphalor/mousewheelie/client/inventory/ContainerScreenHelper.java

```diff
@@ -12,121 +12,132 @@
 import net.minecraft.entity.player.PlayerInventory;
 import net.minecraft.item.ItemStack;
 
+import static net.minecraft.client.gui.screen.Screen.hasAltDown;
```

Comment:
Why do you use a static import?
The class is already imported and static imports should be used only if strictly necessary.

Reply:
it fixed in new commit